requirements.txt

```diff
@@ -1,15 +1,11 @@
-braintree==3.57.1
-frappe
```

Comment:
Probably leave `frappe` as a dependency still? At least for the sake of correctness

Reply:
Alright, re-added it